locales/base/translation.json

```diff
@@ -2568,11 +2568,13 @@
     "depositBottomSheet": {
       "title": "Review & Deposit",
       "description": "Before depositing to an Aave pool, please review the transaction details below",
+      "descriptionV1_93": "Before depositing to a {{providerName}} pool, please review the transaction details below",
```

Comment:
are we okay with it saying "... a Aave pool..." rather than "... an Aave pool..."? Not sure how to prevent this.

Reply:
chatted with John and decided to go with `an` for now for the 2 pools. We can revisit this when we add more pools and should be something we can fix OTA